packages/components/src/components/post-back-to-top/post-back-to-top.scss

```
@@ -0,0 +1,46 @@
+@use '@swisspost/design-system-styles/tokens/components';
+@use '@swisspost/design-system-styles/functions/tokens';
+@use '@swisspost/design-system-styles/components/button';
+@use '@swisspost/design-system-styles/core' as post;
+@use '@swisspost/design-system-styles/mixins/icons' as icons;
+@use '@swisspost/design-system-styles/mixins/utilities';
+@use '@swisspost/design-system-styles/mixins/elevation' as elevation-mx;
```

You can either import all the mixins individually:
```suggestion
@use '@swisspost/design-system-styles/mixins/button' as button-mx;
@use '@swisspost/design-system-styles/mixins/icons' as icons;
@use '@swisspost/design-system-styles/mixins/utilities';
@use '@swisspost/design-system-styles/mixins/elevation' as elevation-mx;
```

Or use the core file only: 
```suggestion
@use '@swisspost/design-system-styles/core' as post;
```